src/app/dashboard/dashboard.test.js

```diff
@@ -39,4 +39,20 @@ describe('Dashboard', () => {
             expect(sut.executors).toEqual([executor]);
         });
     });
+
+    describe('isDashboardEmpty', () => {
+        it('should return \'true\' if there is no sensors and executors', () => {
```

Comment:
May be better name of the test is 
'should dashboard be is empty when there is no sensors and executors'

to  exclude tech details and call our tests according business logic